content/r/concepts/csv-files/terms/ncol/ncol.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'ncol()'
+Description: 'Returns the number of columns in a data frame.'
```

Comment:
Again, this doesn't seem particular to CSV objects, and should go under a data frame entry.